StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+PaymentMethodAvailability.swift

```diff
@@ -23,6 +23,7 @@ extension PaymentSheet {
         .klarna, .afterpayClearpay, .affirm,
         .iDEAL, .bancontact, .sofort, .SEPADebit, .EPS, .giropay, .przelewy24,
         .USBankAccount,
+        .instantDebits,
```

Comment:
This was necessary because we now do extra filtering of `instantDebits`

Code point 1:
- https://github.com/stripe/stripe-ios/blob/ef830fd8/StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift#L210

Code point 2:
- https://github.com/stripe/stripe-ios/blob/ef830fd8/StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift#L336-L338